### DISCUSSION TITLE
is hardware acceleration igpu or cpu ?

### DISCUSSION THREAD

**Discussion by peter247** [blakeblackshear/frigate#2696] [General]
Now my Deskmini x300 with a ryzen 5700g and proxmox can use igpu passthrough because before they was all in the same group .
When you use hardware acceleration for transcoding in frigate etc , are you using the cpu or igpu ?
Because if it is the igpu I guess I can't give it to a VM if I want to use hardware acceleration.

**Comment by NickM-27** [comment]
CPU is used by default, acceleration would be using a GPU / other device that can handle decoding more quickly than a general purpose CPU. I have a ryzen 3200G that I use in unRAID docker as hardware acceleration, it works well. 

It seems true it can only be passed / used for one VM / env at a time
